lib/core/test/unit/Cardano/Wallet/Primitive/BlockSummarySpec.hs

```diff
@@ -28,32 +32,58 @@ import Test.QuickCheck
     ( Arbitrary (..)
     , Gen
     , Property
+    , forAll
     , listOf1
     , property
     , resize
     , shuffle
+    , sublistOf
     , (===)
     )
 
 import Data.Map.Strict as Map
 
 spec :: Spec
 spec = do
+    describe "Sublist" $ do
+        it "merging is idempotent" $
+            property prop_idempotent
+        it "merging has whole list as neutral element" $
```

Comment:
if we have `I=All` and `A=Some` then `merge I A = I` is `I` neutral element really?